packages/tailwindcss/src/utilities.test.ts

```diff
@@ -21239,6 +21301,11 @@ test('shadow', async () => {
       box-shadow: var(--tw-inset-shadow), var(--tw-inset-ring-shadow), var(--tw-ring-offset-shadow), var(--tw-ring-shadow), var(--tw-shadow);
     }
 
+    .shadow-blue-500 {
+      --tw-shadow: #3b82f6;
+      box-shadow: var(--tw-inset-shadow), var(--tw-inset-ring-shadow), var(--tw-ring-offset-shadow), var(--tw-ring-shadow), var(--tw-shadow);
+    }
```

Comment:
This seems wrong, this ins setting the shadow property and not the shadow-color here. Looks like you want to define this as `--box-shadow-color-blue-500`.

Reply:
Interesting, now my tests are passing locally, but are failing on CI 🤔

Reply:
@scottbedard Haha yep sorry!

Reply:
Thanks! Our team is excited to really polish our design system and modernize with Tailwind ❤️